webpack.config.js

```diff
@@ -39,36 +27,25 @@ module.exports = function(env, argv) {
                 {
                     test: /\.js$/,
                     include: [
-                        path.resolve(__dirname, "src"),
-                        path.resolve(__dirname, "node_modules/debug")
+                        path.join(__dirname, "src"),
+                        require.resolve("debug")
```

Comment:
This does not work properly. We need to tell Babel to transpile the entire `debug` source and not just the entry file.
Otherwise [some ES6 code](https://github.com/kherock/client-js/blob/master/dist/build/fhir-client.js#L7778-L7784) will leak into to bundle and break in IE.

Reply:
That makes sense. When I saw `node_modules/` I assumed it was intending to match relative to the project workspace.

Reply:
Actually looking at this again, my code is equivalent to the previous revision!
```
Welcome to Node.js v12.13.0.
Type ".help" for more information.
> __dirname = process.cwd()
'/home/smart-on-fhir/client-js'
> path.resolve(__dirname, "node_modules/debug")
'/home/smart-on-fhir/client-js/node_modules/debug'
> path.resolve(__dirname, "node_modules/debug")
'/home/smart-on-fhir/client-js/node_modules/debug'
> 
```

If you want the behavior of including all versions in the hierarchy, `__dirname` shouldn't be there.

Reply:
Also, core-js doesn't provide a console polyfill at all, so the line in question doesn't actually do anything. If you want to avoid surprises with console on IE in standards mode, you need your own polyfill, or something like [ie-console-patch](https://www.npmjs.com/package/ie-console-patch).